### PR TITLE
Username mentioned in user profile HTML

### PR DESCRIPTION
**Pull request by davidmz**

### CODE REVIEW

File: public/js/app/templates/profileLinksTemplate.handlebars [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div class="profile p-user-profile">
+<div class="profile p-user-profile" data-username="{{ content.user.username }}">
```
Comment (clbn) [inline public/js/app/templates/profileLinksTemplate.handlebars:1]: Oh noes! I see spaces in the braces!

Comment (berkus) [inline public/js/app/templates/profileLinksTemplate.handlebars:1]: They are insignificant, but I will fix for consistency.

Comment (berkus) [inline public/js/app/templates/profileLinksTemplate.handlebars:1]: In [ffb80ce]

Comment (clbn) [inline public/js/app/templates/profileLinksTemplate.handlebars:1]: @berkus Nah, don't mind, I'm just nitpicking.

Comment (clbn) [inline public/js/app/templates/profileLinksTemplate.handlebars:1]: @berkus Thanks for fixing it!

**Pull request merged**